config/cesm/machines/config_compilers.xml

```diff
@@ -200,7 +200,7 @@ using a fortran linker.
   <FFLAGS>
     <!-- -ffree-line-length-none and -ffixed-line-length-none need to be in FFLAGS rather than in FIXEDFLAGS/FREEFLAGS
        so that these are passed to cmake builds (cmake builds don't use FIXEDFLAGS and FREEFLAGS). -->
-    <base>  -fconvert=big-endian -ffree-line-length-none -ffixed-line-length-none </base>
+    <base>  -fconvert=big-endian -ffree-line-length-none -ffixed-line-length-none -fallow-argument-mismatch -fallow-invalid-boz </base>
```

Comment:
Thanks for adding these flags. But, since they are workarounds for issues that will hopefully be resolved, I think it would be helpful to add a comment stating that – i.e., something like:

```
-fallow-argument-mismatch is needed for building with the mpich library with gfortran10; -fallow-invalid-boz is needed for some components that have non-standard uses of BOZ literal constants with gfortran10. See https://github.com/ESMCI/cime/issues/3846 for details.
```

However, I'm also wondering if these flags work with older versions of gfortran, or if they require gfortran10+.